server/service/src/sync/translations/invoice_line.rs

```diff
@@ -371,17 +380,17 @@ impl SyncTranslation for InvoiceLineTranslation {
                     vvm_status_id,
                     reason_option_id,
                     campaign_id,
+                    program_id,
                     shipped_number_of_packs,
                 },
             item_row,
             ..
         } = invoice_line;
 
-        let oms_fields = if campaign_id.is_some() {
```

Comment:
Removed the is_some() check, otherwise we wouldn't sync if we changes from a Some() to a None.

Does mean that oms fields will now always be Some() ... could check each field, set to None if every field is None, but that seemed like overkill